wallet/wallet-node-client/src/rpc_client/mod.rs

```diff
@@ -33,6 +37,21 @@ pub enum NodeRpcError {
     ResponseError(jsonrpsee::core::ClientError),
 }
 
+#[derive(Clone, Debug)]
+pub struct MaybeDummyNode {
+    opt_client: Option<NodeRpcClient>,
```

Comment:
Why do we need a "MaybeDummyNode" and an Option? Why don't we create an RPC client that's only for cold wallet?